.openshift-ci/build/build-collector.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+export SRC_ROOT_DIR=$1
+export DISABLE_PROFILING="true"
+export CMAKE_BUILD_DIR="$SRC_ROOT_DIR/cmake-build"
+
+make -C $SRC_ROOT_DIR/collector pre-build
+/build-collector.sh
```

Comment:
You need to update the path here to `$SRC_ROOT_DIR/builder/build/build-collector.sh`

Reply:
I don't think I need to since build-collector.sh is copied to the root directory, but I don't think your change is bad, so I made the change.